api/api/Roles/models.py

```diff
@@ -0,0 +1,22 @@
+from accounts.models import User
+from clubs.models import Club
+from django.db import models
+
+
+class Roles(models.Model):
+
+    CLUB_ROLE_TYPES = [
+        ('Coordinator', 'Coordinator'),
+        ('Co-Coordinator', 'Co-Coordinator'),
+        ('Faculty Incharge', 'Faculty Incharge'),
+        ('Convener', 'Convener'),
+        ('Co-convener', 'Co-convener'),
+        ('Conselor', 'Conselor'),
+        ('Associate-Conselor', 'Associate-Conselor')
+    ]
```

Comment:
We should also add a role for core members as it is not necessary that only the coco or the coordinator will host an event.